public/redux/reducers/alerting.ts

```diff
@@ -84,6 +85,18 @@ const reducer = handleActions<Monitors>(
         errorMessage: action.error,
       }),
     },
+
+    [SEARCH_ALERTS]: {
+      REQUEST: (state: Monitors): Monitors => ({
+        ...state
+      }),
+      SUCCESS: (state: Monitors, action: APIResponseAction): Monitors => ({
+        ...state
+      }),
+      FAILURE: (state: Monitors, action: APIResponseAction): Monitors => ({
+        ...state
+      }),
+    },
```

Comment:
Maybe don't need to store alert info in redux store, but can you at least update the `requesting` and `errorMessage` fields? Can be helpful later on when determining loading states and propagating errors

Reply:
Yes, not sure if this will impact other components using `requesting` and `errorMessage`. Will add todo here and add it later.

Reply:
Sure, not a blocker. If anything, adding `requesting` and `errorMessage` wouldn't hurt, as it would just make the frontend more accurate.

Reply:
Added toto here, if any client components using these search APIs needs `requesting` and `errorMessage`, feel free to resolve this todo..